public/components/visualize/components/sample-data.test.js

```diff
@@ -0,0 +1,60 @@
+import { SampleData } from './sample-data';
+import { WzRequest } from '../../../react-services';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import { getErrorOrchestrator } from '../../../react-services/common-services';
+import { UI_ERROR_SEVERITIES } from '../../../react-services/error-orchestrator/types';
+import { UI_LOGGER_LEVELS } from '../../../../common/constants';
+import React from 'react';
+
+const awaitForMyComponent = async (wrapper) => {
+  await act(async () => {
+    await new Promise((resolve) => setTimeout(resolve, 0));
+    wrapper.update();
+  });
+};
+jest.mock('../../../react-services');
+jest.mock('../../../react-services/common-services');
+describe('Check sample data component', () => {
+  it('should render if there is sample data', async () => {
+    WzRequest.genericReq.mockResolvedValue({ data: { sampleAlertsInstalled: true } });
+    const wrapper = await mount(<SampleData />);
+    await awaitForMyComponent(wrapper);
+    expect(wrapper.find('EuiCallOut').exists());
```

Comment:
Here you can add some `expect` findings for your `title="This dashboard contains sample data"`
Great job!

Reply:
Added an extra test checking for the prop